src/covid_model_seiir_pipeline/forecasting/workflow.py

```diff
@@ -5,6 +5,27 @@
 
 from covid_model_seiir_pipeline.workflow_template import TaskTemplate, WorkflowTemplate
 
+FORECAST_RUNTIME = 1000
+FORECAST_MEMORY = '5G'
+FORECAST_CORES = 1
+FORECAST_SCALING_CORES = 50
+FORECAST_QUEUE = 'd.q'
```

Comment:
Consider pulling these from environment variables, using the above values as defaults

I've never been happy in the long-run when I have to update code to use a less busy queue